### ISSUE TITLE
.Some() / .Every() not working on recursive relations

### ISSUE THREAD

**Issue by maaft** [prisma/prisma-client-go#206]
Schema:
```
datasource db {
    provider = "sqlite"
    url      = "file:dev.db"
}

generator db {
    provider = "go run github.com/prisma/prisma-client-go"
    output = "./db/db_gen.go"
    package = "db"
}

model User {
    id        String   @default(cuid()) @id

    posts     Post[]
}

model Post {
    id        String   @default(cuid()) @id
    title     String

    morePosts Post[]
}
```

```golang
        post, err := client.Post.CreateOne(
		db.Post.Title.Set("hello"),
	).Exec(ctx)

	_, err = client.Post.CreateOne(
		db.Post.Title.Set("my"),
		db.Post.Post.Link(db.Post.ID.Equals(post.ID)),
	).Exec(ctx)

        _, err = client.Post.FindMany(
		db.Post.MorePosts.Every(
                     db.Post.Title.Contains("my"),
		),
	).Exec(ctx)
```

Error:
panic: Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(SqliteFailure(Error { code: Unknown, extended_code: 1 }, Some("near \")\": syntax error"))) })

**Comment by steebchen** [comment]
opened an issue at https://github.com/prisma/prisma/issues/3354

**Comment by steebchen** [comment]
This should be fixed in [v0.0.9](https://github.com/prisma/prisma-client-go/releases/tag/v0.0.9).
